bootloader/test_infra/README.md

```diff
@@ -0,0 +1,15 @@
+# Testing infrastructure for bootloader
+
+This crate allows you to run the unittests against the bootloader code.
+
+You should put your tests in ../tests/bootloader/bootloader_test.yul, then compile the yul with:
+
+```shell
+yarn preprocess && yarn compile-yul
```

Comment:
Btw, system contracts should be compiled as well, because artifacts are used in the preprocessing

Reply:
good point - thanks.